### DISCUSSION TITLE
macOS 15 Sequoia

### DISCUSSION THREAD

**Discussion by SkyrilHD** [SkyrilHD/Dell-Exx50-Hackintosh#78] [Show and tell]
Same as last year, the newest beta boots on my E7250. However, due to the lack of GPU acceleration, it is quite unusable.

![Bildschirmfoto 2024-06-15 um 06 55 52](https://github.com/SkyrilHD/Dell-E7250-Hackintosh/assets/28839925/054d8a71-e814-449f-b804-e5b252c7170a)

**Comment by SkyrilHD** [comment]
Now with the early preview of OCLP, we finally have GPU acceleration working.

![Screenshot 2024-08-22 at 14 06 05](https://github.com/user-attachments/assets/6cea2a97-8b19-4992-a473-ab878d642022)

**Comment by mian196** [comment]
Hi, which version of efi i should use to install macOS Sonoma or ventura as i have some graphics issue on sonoma so i want to stay on ventura right now. and thanks for making this repo helps a lot. And it should be good to create a telegram channel or discord sever for discussion if you think this is good idea.

**Reply by SkyrilHD** [reply]
You can use the Sequoia EFI on Ventura (which I would recommend due to newer kexts and OpenCore). In your case you will need to replace AirportItlwm with a Ventura compatible one, as we use the Sonoma version of AirportItlwm.

Basically, the EFI release is built like this:
- Monterey: Can boot from Yosemite to Monterey without issues (Catalina and older need changing some values in secure boot and APFS).
- Sequoia: Ventura - latest

The only changes you need to make when upgrading or downgrading any major version is to replace AirportItlwm if you want to use a native Intel implementation. Otherwise, it is easier to use Heliport when dual-booting different OSes.

For various reasons, I will not create a channel or server. One reason is that I want to keep everything together to avoid confusion for myself and other users. Preferably, users should create a discussion page here. Or they can post in this thread. Alternatively, creating an issue ticket is also possible (and actually the best way to contact me since I get notifications).

**Comment by Ronald-Hoo** [comment]
Hi Skyril,

Used your EFI to install Sequoia on a Latitude E7450, runs perfectly. There is one 'problem', the CPU gets really hot, even in idle state it gets to 60ºC. Do you have any tips/ideas to fix this? Thanks!

**Reply by SkyrilHD** [reply]
I noticed there are a lot of questions why the CPU is getting hot. My guess is that newer macOS operating systems are more demanding on the CPU. At the same time, the CPU is getting older. You may want to try applying a new thermal paste. It has been 9 years since the laptop was released. Also keep OCLP up to date, as they contain bug fixes and performance improvements.

For basic troubleshooting steps: 
https://github.com/SkyrilHD/Dell-E7250-Hackintosh/issues/79#issuecomment-1902632648

**Reply by Ronald-Hoo** [reply]
Thanks!

**Comment by KCVEMURI** [comment]
I have installed Mac Montery on E5450 few years back and now wanted to upgrade to Sequoia. I tried to copy the EFI folder into EFI parition for sequoia but when i try to check for software upgrade it doesn't show any upgrade. which EFI file should i use so that i can upgrade from Monterey to Sequoia?

**Reply by SkyrilHD** [reply]
The way I would I upgrade is as follows:
1. Download macOS Sequoia Installer (you can use tools like gibmacOS or OpenCore Legacy Patcher)
2. Download OpenCore Legacy Installer (mandatory for patching GPU!)
3. Put the Sequoia EFI to a USB stick (FAT32; GUID partition table) and boot Monterey **from the USB**!
4. You should notice you lose WiFi (This means you booted the correct EFI)
5. Install macOS Sequoia from the previously downloaded installer.
6. Use your USB's EFI until you reach the desktop.
7. Install Root patches using OpenCore Legacy Patcher
8. Replace the internal's EFI with one from the USB

The reason why you download beforehand is due to losing WiFi when using the Sequoia EFI.
To fix WiFi on Sequoia, replace AirportItlwm with [itlwm](https://github.com/OpenIntelWireless/itlwm/releases) and download [Heliport](https://github.com/OpenIntelWireless/HeliPort/releases/tag/v2.0.0-alpha). Do not forget to update the config as well!
Also do not forget to update the SMBIOS when replacing the EFI.

**Reply by KCVEMURI** [reply]
Thanks for the reply, I am new to this update as i did it long back. I don't understand how you boot monterey from USB when i have loaded USB with only Sequioa EFI? The way i was trying is to take backup of the Monterey EFI and update the BOOT and OC folders in EFI partition with Sequioa and once the installer download is complete install it . But when i tried to install it i got a stop error indicating that kerner has panicked. I have recovered it back to montery using backup EFI for now Can u please elaborate step 3-7 bit more detail so that i can do it correctly. thanks for the help

Also what updates i need to do to config and SMBIOS ?

**Reply by SkyrilHD** [reply]
With the Sequoia EFI you can still boot to any macOS, however the security is reduced to make patching GPU work. You use the Sequoia EFI to unlock booting and installing Sequoia. The reason I would keep the internal EFI is to prevent not booting at all, because you know the EFI is working. Using a USB helps you to update without breaking your original EFI.

Step 3: Format any USB as FAT32 and GUID partition table (GPT). Copy the Sequoia EFI the root of the USB (Do not mount the EFI partition of the USB). Reboot the laptop and boot to the USB via Boot Manager, then select your Monterey install. If you cannot boot into Monterey, do not go any further.
Step 4: When you boot into Monterey with Sequoia's EFI, you will lose WiFi, security (SIP). However this enables updating to Sequoia.
Step 5: Run the `Install macOS Sequoia.app` to install Sequoia. Follow the instructions.
Step 6: On reboot, boot to the USB again. There will be a new partition called "macOS Installer" (or similar to that). Boot into that, and let the installer do its thing. There will be multiple reboots and select "macOS Installer" as long as you see it. On final reboot the "macOS Installer" should disappear, and you can boot to your "normal" partition. Go through the setup until you reach the desktop. **The setup will be slow due to no acceleration, and that's normal!** 
Step 7: You use OpenCore Legacy Patcher to fix the GPU acceleration by installing the Root Patches. After patching and reboot, the performance should be normal again.

As for WiFi on Sequoia, download itlwm.kext from the link above and in the EFI/OC/Kexts delete AirportItlwm.kext and add itlwm.kext. In the config, you need to replace "AirportItlwm" with "itlwm" to enable itlwm. Otherwise you will still have no WiFi. Since you added itlwm, you need to use HeliPort (link above) to connect to any network. This app replaces the native WiFi UI. The native WiFi UI does not work on Sequoia+.
